pkg/arm/v6/data/master-startup.sh

```diff
@@ -1,5 +1,54 @@
 #!/bin/bash -ex
 
+{{ if gt (len .Config.SecurityPatchPackages) 0 }}
+logger -t master-startup.sh "installing red hat cdn configuration on $(hostname)"
+cat >/var/lib/yum/client-cert.pem <<'EOF'
+{{ CertAsBytes .Config.Certificates.PackageRepository.Cert | String }}
+EOF
+cat >/var/lib/yum/client-key.pem <<'EOF'
+{{ PrivateKeyAsBytes .Config.Certificates.PackageRepository.Key | String }}
+EOF
+# TODO: delete the following section after all clusters are run with a vm image with kickstart.repo baked in
+cat > /etc/yum.repos.d/kickstart.repo <<'EOF'
+[rhel-7-server-rpms]
+name=Red Hat Enterprise Linux 7 Server (RPMs)
+baseurl=https://cdn.redhat.com/content/dist/rhel/server/7/7Server/$basearch/os
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+sslcacert=/etc/rhsm/ca/redhat-uep.pem
+sslclientcert=/var/lib/yum/client-cert.pem
+sslclientkey=/var/lib/yum/client-key.pem
+enabled=yes
+
+[rhel-7-server-extras-rpms]
+name=Red Hat Enterprise Linux 7 Server - Extras (RPMs)
+baseurl=https://cdn.redhat.com/content/dist/rhel/server/7/7Server/$basearch/extras/os
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+sslcacert=/etc/rhsm/ca/redhat-uep.pem
+sslclientcert=/var/lib/yum/client-cert.pem
+sslclientkey=/var/lib/yum/client-key.pem
+enabled=yes
+
+[rhel-7-server-ose-3.11-rpms]
+name=Red Hat OpenShift Container Platform 3.11 (RPMs)
+baseurl=https://cdn.redhat.com/content/dist/rhel/server/7/7Server/$basearch/ose/3.11/os
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+sslcacert=/etc/rhsm/ca/redhat-uep.pem
+sslclientcert=/var/lib/yum/client-cert.pem
+sslclientkey=/var/lib/yum/client-key.pem
+enabled=yes
+EOF
+yum makecache
+
+{{ range .Config.SecurityPatchPackages }}
+logger -t master-startup.sh "installing {{.}} on $(hostname)"
+yum install -y {{.}}
```

Comment:
need to confirm install/update @ehashman 
I think it's preferable to construct a single yum command line and run it once rather than range

Reply:
I think we want to use `yum update`.

Also agree that it's preferable to specify this as a single yum command with one package rather than a range---less chance for failure and lets the yum dep resolver run just once.

Reply:
I think i'm going to use `yum install` here because:
- yum is smart enough that when you call install for a package which is already on the system, an update is done
```
[root@master-000001 cloud-user]# yum install -y python-libs-2.7.5-80.el7_6.x86_64
...
Updated:
  python-libs.x86_64 0:2.7.5-80.el7_6 

Dependency Updated: 
  python.x86_64 0:2.7.5-80.el7_6  

Complete!
```
- if you run `yum update` on a non-existent package, exit code 0 is returned. `yum install` returns the expected (imo) exit code 1 and this will make error handling more straightforward.
```
[root@master-000001 cloud-user]# yum update -y -q hell
No Match for argument: hell
[root@master-000001 cloud-user]# echo $?
0
[root@master-000001 cloud-user]# yum install -y -q hell
Error: Nothing to do
[root@master-000001 cloud-user]# echo $?
1
```
I'll join the packages and install them with a single `yum install`. Also adding a `--quiet` flag to the command.

Reply:
The risk of using `yum install` is that if a package is specified that isn't already installed on the system, it will go ahead and install that package, which is not a risk with `yum update`.

I don't think it's necessarily a problem that yum update returns exit code 0 on a nonexistent package, but would using `yum update-to` here instead fix that?